spec.bs

```diff
@@ -3474,6 +3474,10 @@ Each {{InterestGroupReportingScriptRunnerGlobalScope}} has a
 <div algorithm>
   The <dfn method for="InterestGroupReportingScriptRunnerGlobalScope">registerAdMacro(|name|,
   |value|)</dfn> method steps are:
+  1. [=list/For each=] [=code unit=] |c| in |name| and |value|:
+     1. [=exception/Throw=] a {{TypeError}} if both of the following conditions hold:

```

Comment:
since there are only two conditions, maybe easier and more readable to just say "if |c| is not ... and |c| is not `%`"

Reply:
I prefer this way since it gets difficult to read the boundaries of the conjunctions in English, especially when they include "not" inside them

Reply:
ack.